_content/articles/our-reaction-to-dukkha_ashby-elizabeth.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Our Reaction to Dukkha"
+authors:
+  - "Elizabeth Ashby"
+external_url: "https://www.bps.lk/olib/bl/bl026_Ashby_Our-Reactions-to-Dukkha.pdf"
```

Comment:
Use the html version for the external_url of wheels and bodhi leaves: https://www.bps.lk/olib/bl/bl026_Ashby_Our-Reactions-to-Dukkha.html